clayui.com/src/templates/docs.js

```diff
@@ -465,5 +492,8 @@ export const pageQuery = graphql`
 		mdxTab: mdx(fields: {slug: {ne: "", eq: $sibling}}) {
 			body
 		}
+		apiTab: mdx(fields: {slug: {ne: "", eq: $sibling}}) {
```

Comment:
Should this be `$siblings`? (plural)

Reply:
also since siblings is an array, you'll have to loop up the graphql query for that since `eq` expects a string I think